textproc/libstemmer/Portfile

```diff
@@ -3,31 +3,31 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        snowballstem snowball 2a3f28097af3651f0c1c6da8998d676b487d5651
+github.setup        snowballstem snowball 2.0.0 v
```

Comment:
I'm guessing that this will require invoking the `epoch` line.

Reply:
probably not as the version was previously set to ```0.1-20191013``` (see line 13 below)

Reply:
Oh yes you are right. I didn't see it because the version field doesn't usually appear there.